### DISCUSSION TITLE
Yale YMF40 door lock failed interview

### DISCUSSION THREAD

**Discussion by EvertonAdroit** [Koenkk/zigbee2mqtt#11710] [Devices]
Hey everybody. Can someone help me with something that it's causing me trouble for weeks now..
I have 2 Yale door locks.. YMF40 and YRL226L. The 226 paired and it is working flawless. The problem is with the 40...
Every time I try to pair it, the pair kinda works.. The lock appears in the zigbee2mqtt dashboard, but with a red exclamation in the icon.

It won't show battery level.

And it won't send anything when it's unlocked via fingerprint, but when it's unlocked via HA or password, it works normally
I even tried opening a github issue in the zigbee2mqtt repository, but got no response after weeks
I use the new sonoff 3.0 zigbee hub as coordinator and 2 more as routers, but the coordinator is the closest to the lock

this is the error that appears when I try to pair it:

Error Failed to configure 'Fechadura Sala', attempt 1 (Error: Bind 0x000d6f0016fdc36f/1 closuresDoorLock from '0x00124b00256c5557/1' failed (AREQ - ZDO - bindRsp after 10000ms) at Timeout._onTimeout (/app/node_modules/zigbee-herdsman/src/utils/waitress.ts:64:35) at listOnTimeout (internal/timers.js:557:17) at processTimers (internal/timers.js:500:7))
Error Failed to interview 'Fechadura Sala', device has not successfully been paired
Error Failed to configure 'Fechadura Sala', attempt 2 (Error: Bind 0x000d6f0016fdc36f/1 genPowerCfg from '0x00124b00256c5557/1' failed (AREQ - ZDO - bindRsp after 10000ms) at Timeout._onTimeout (/app/node_modules/zigbee-herdsman/src/utils/waitress.ts:64:35) at listOnTimeout (internal/timers.js:557:17) at processTimers (internal/timers.js:500:7))

I've already tried removing the batteries and starting again. Got it very close to the coordinator and the router (tried without the router connected as well). Basically tried every tips from here https://www.zigbee2mqtt.io/guide/faq/ with no success.

Would really appreciate any help with this.

**Comment by gregolin** [comment]
Hi Everton.
Any news regarding this issue? I have almost the same scenario than you (2 different lockers, 1 paired with success and other one with interview issue - YMF40).
Thanks.

**Reply by EvertonAdroit** [reply]
No news.. tried other forums and discord servers and got nothing...

**Comment by EvertonCa** [comment]
This issue remains until today. Anyone could help with this?
